package.json

```diff
@@ -92,6 +92,7 @@
 		"@automattic/explat-client-react-helpers": "0.0.2",
 		"@woocommerce/e2e-environment": "0.2.2",
 		"@woocommerce/e2e-utils": "0.1.5",
+		"@woocommerce/settings": "^1.0.0",
```

Comment:
Need for the `isWpVersion` function

Reply:
Weird that this function is in `@woocomerce/settings` 🤔

Reply:
I thought so to, maybe I will have another look to see if it's available anywhere else.